.github/workflows/cicd.yml

```diff
@@ -1,5 +1,10 @@
 name: CI/CD
-on: push
+on:
+  push:
+    tags-ignore:
+      - '*.*'
+    paths-ignore:
+      - package.json
```

Comment:
To prevent unnecessary jobs from being triggered by pushing bump version commit with admin github token.

Below is the experiment result in my forked repo:
**Before:**
![image](https://user-images.githubusercontent.com/13249748/100566376-c8fd6b80-3300-11eb-9707-962353ec1226.png)
**After:**
![image](https://user-images.githubusercontent.com/13249748/100566445-fea25480-3300-11eb-976f-a7e3114c001e.png)